README.md

```diff
@@ -25,6 +24,7 @@ In light of recent developments, we have introduced a series of RPMs web develop
 Kubernetes (commonly referred to as k8s) has emerged as the de-facto standard for orchestrating containers, enabling the large-scale management and configuration of containers as cattle rather than pets. OpenShift is a Platform-as-a-Service (PaaS) built on top of Kubernetes, providing comprehensive application lifecycle management. This includes the ability to quickly build and deploy Docker-formatted containers while managing them on a robust, scalable platform. To learn more about these two technologies, explore the following resources:
 
 - https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/#kubernetes-is
+-- FIX URL FOR RHEL 8 or later --
 - https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html-single/getting_started_with_kubernetes/
```

Comment:
@phracek 
I can find this for later RHELs. What's the correct current URL?

Reply:
This can be removed. Collection reached EOL.